lib/husky_musher/utils/redcap.py

```diff
@@ -279,11 +285,16 @@ def max_instance(instrument: str, redcap_record: List[dict], since: int,
     events_instrument_complete = [
         encounter
         for encounter in redcap_record
-        if int(encounter['redcap_repeat_instance']) >= since
-        and encounter[f"{instrument}_complete"] != ''
+        if encounter[f"{instrument}_complete"] != ''
         and is_complete(instrument, encounter) == complete
     ]
 
+    if since is not None:
+        events_instrument_complete = list(filter(
+            lambda encounter: int(encounter['redcap_repeat_instance']) >= since,
+            events_instrument_complete
+        ))
```

Comment:
Do you need an else statement here to filter to instances since one week ago if `since` is None? You might return a max instance that is from earlier than a week ago.

Reply:
Heh, that was my initial thought too until I realized at this point we're only dealing with encounters in the past week. See [this upstream function](https://github.com/seattleflu/husky-musher/blob/2e0c84ba4b1b33280b4a891d8a05874053f1b4bb/lib/husky_musher/utils/redcap.py#L162).

Reply:
Ahh great that makes sense. Maybe we should have a comment here to remind us of that so this question doesn't come up again 😄

Reply:
Great point. Added in a new commit.